compliance_checker/tests/test_base.py

```diff
@@ -81,3 +81,18 @@ def verify_dummy(ds):
         self.ds.dummy = 'dummy'
         base.attr_check(attr, self.ds, priority, rv3)
         assert rv3[0] == base.Result(priority, True, 'dummy', [])
+
+
+class TestGenericFile(TestCase):
+    '''
+    Tests the GenericFile class.
+    '''
+
+    def test_create_GenericFile_success(self):
+        path = '/tmp/test.txt'
```

Comment:
@agstephens this is just testing the pathname not opening a file, right? Otherwise we'll need to support Windows path names too (os.path.join).

Reply:
Hi @Bobfrat, that is correct. `GenericFile` does not look inside the file. It only looks at it from outside. So it might: 1. Check directory and file name match fixed rules, 2. Get size of the file., etc.